apps/i18n/fish/fr_fr.json

```diff
@@ -2,24 +2,24 @@
   "fishvtrash-training-init1": "Les déchets accumulés dans l'eau ont un impact sur la vie marine. Dans cette activité, tu vas programmer ou entraîner une I.A. (Intelligence Artificielle) à identifier des poissons ou des déchets. Viens, on va nettoyer l'océan ! Clique n'importe où sur l'écran pour continuer.",
   "fishvtrash-training-init2": "Allons rencontrer l'I.A.",
   "fishvtrash-training-init3": "L’ I.A. ne sait pas si un objet est un poisson ou un déchet, mais elle peut analyser des images et identifier des motifs.",
-  "fishvtrash-training-init4": "Pour programmer l' I.A., utilisez les boutons pour désigner une image comme \"Poisson\" ou \"Pas un poisson\". L'entraînement que vous donnez à l' I.A. lui permettra de reconnaître des motifs par elle-même. Commençons!",
+  "fishvtrash-training-init4": "Pour programmer l' I.A., utilisez les boutons pour désigner une image comme « Poisson » ou « Pas un poisson ». L'entraînement que vous donnez à l' I.A. lui permettra de reconnaître des motifs par elle-même. Commençons!",
```

Comment:
More `« `'s in this commit as well.

Reply:
It seems to work.
<img width="1483" alt="Screenshot 2023-01-10 at 11 21 38" src="https://user-images.githubusercontent.com/66776217/211619484-f7e986f4-6720-42c6-8caf-53bc0eec68f7.png">

Reply:
Also tested in safary and works too.